selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/ImportWizardFormTest.java

```diff
@@ -143,6 +146,11 @@ private void deletePrivateSshKey() throws Exception {
     preferences.closeForm();
   }
 
+  @AfterMethod
+  private void deleteProject() throws Exception {
+    testProjectServiceClient.deleteResource(ws.getId(), currentProjectName);
```

Comment:
IMHO it's better to ignore possible exception here if required resource doesn't exist.